src/targets/services/aggregation.js

```diff
@@ -2,121 +2,71 @@ global.fetch = require('node-fetch').default
 global.btoa = require('btoa')
 
 import fs from 'fs'
-import CozyClient from 'cozy-client'
+import CozyClient, { Q } from 'cozy-client'
 import { Model } from './helpers'
 import dissecConfig from '../../../dissec.config.json'
 
 export const aggregation = async () => {
+  const client = CozyClient.fromEnv(process.env, {})
+
+  var originalConsoleLog = console.log
+  console.log = function () {
+    let args = []
+    args.push('[' + client.stackClient.uri.split('/')[2] + '] ')
+    for (var i = 0; i < arguments.length; i++) {
+      args.push(arguments[i])
+    }
+    originalConsoleLog.apply(console, args)
+  }
```

Comment:
This is useful but shouldn't be here, maybe export a `logWithInstance` function in a debug.js file?